### PR TITLE
Badmin Gauntlet Update

### PR DESCRIPTION
**Pull request by ghost**
:cl:
tweak: If there are already 2 cakes, Let There Be Cake! will teleport the most eaten one to the user's feet.
fix: Fixed runtime spam with cluwnes
soundadd: New sound effect for Badmin Gauntlet snap
add: When you put the final stone into the Badmin Gauntlet, there will be a 5 second "full power sequence", before you are able to snap.
fix: The Snap now pulls from a list of living player mobs, rather than all player mobs total. Should result in half the station's living population actually being snapped.
del: Got rid of The Garden. It was ugly and served no purpose.
imageadd: New screen overlay for The Snap.
fix: the Revengers Hulk should now actually be green.
balance: The Revengers (or any ERT, really) can no longer be knocked down by shockwave, and all of them now have Reviver/CNS implants.
fix: Fixed display order of badmin stone/gauntlet examine.
fix: Fixed infinity cake not containing any chemicals.
balance: You can no longer go into blood crawl with the Badmin Stones in your possession, they'll be dropped on the floor.
add: Speaking of blood crawl, if someone is stupid enough to revive thanos with cluwne rise (this HAS happened before), he can now put the stones in the gauntlet without a free hand by clicking on them with HARM intent.
fix: The ghost stone no longer acts an inescapable ghost magnet.
balance: The ghost stone's spiritual fist does more damage per ghost, and has a higher maximum damage (5 damage per ghost, minimum of 3, maximum of 35).
tweak: Syndie Stone holders using Super Jump will no longer accidentally destroy their clothes (this was kinda funny but it makes it near-useless)
tweak: Added titanium to the supply pod drop during wardec.
tweak: Assistants now have a small chance to get the Bluespace Stone (man of steal)
add: When the SNAP happens, if the Revengers have not already been called, they will be called.
tweak: Ghost stone cluwnes are colored now. Gold if raised with the badmin gauntlet, purple if raised with the ghost stone.
fix: Fixed pocket sand being finnicky.
add: As it turns out, the Badmin Gauntlet can overpower even the most powerful of beings. 
/:cl:

### CODE REVIEW

**Comment by Majick1** [comment]
15 seconds is way too long, even 5 seconds is a free kill. All this is going to do is make thanos murderbone everyone before snapping

**Comment by ghost** [comment]
@Majick1 i'll reduce it to 5 seconds, because that's approx how long Thanos yelled for in IW after putting the Soul Stone in his gauntlet.

**Comment by Manly-Dwarf** [comment]
>fix: The Snap now pulls from a list of living player mobs, rather than all player mobs total. Should result in half the station's living population actually being snapped.
>add: When you put the final stone into the Badmin Gauntlet, you will be stunned for 5 seconds, before you are able to snap.
>balance: The Revengers can no longer be knocked down by shockwave, and all of them now have Reviver/CNS implants.

lame
nearly everything else is cool
give the cluwnes some sign so people know which side they're on. thanos cluwnes and other cluwnes are indistinguishable

**Comment by ghost** [comment]
@Manly-Dwarf what's wrong with the revengers buff, though? And the Snap list change?

**Comment by ghost** [comment]
also, i made ghost stone cluwnes colored. gold for thanos, purple for ghost stone.

**Comment by ghost** [comment]
and i changed the 5-second to just a 5-second chargeup that he can move around for

**Comment by carlopault** [comment]
looks good to me

**Comment by ghost** [comment]
i REALLY need to change the clown stone intents to something less annoying but i don't know what

**Comment by cb583** [comment]
Shilled in OOC

**Comment by Fikou** [comment]
can you add that ghost stone not being a ghost magnet to spirit blade or something 
spirit blade in general is a shit and needs big fixes or removal

**Comment by ATH1909** [comment]
`	desc = "Summon a powerful cake at your feet, capable of healing those who eat it, and injuring those who are hit by it. <b>Only 2 cakes can exist at the same time.</span>"`
Hey, I found a description that's missing a </b> in the clown stone's file; could you fix it in this PR too?

**Comment by Manly-Dwarf** [comment]
the snap wiping out the station was kinda cool
final stone stun is garbage, I get that its a reference to the film but it just means thanos will run away before putting the stone in or else risk getting insta'd. I doubt you have something that explains that to players, too, so they might rush in, drop the stone, and get themselves killed at the last second.
too many things bypass shockwave now, make it stop. the revengers need something more than just lame stun invulnerability

**Comment by ghost** [comment]
@Manly-Dwarf 
> the snap wiping out the station was kinda cool

It's boring.

> final stone stun is garbage, I get that its a reference to the film but it just means thanos will run away before putting the stone in or else risk getting insta'd. I doubt you have something that explains that to players, too, so they might rush in, drop the stone, and get themselves killed at the last second.

I just replaced it with a non-stunning chargeup sequence where you flash pretty colors faster.

> too many things bypass shockwave now, make it stop. the revengers need something more than just lame stun invulnerability

how are magboots and ERT boots "too many things"? If you have a better idea on how to balance this out, i'm all ears.

**Comment by ghost** [comment]
needed: someone good with photoshop or gimp or whatever to make an animation where the nuke gets snapped away (like the fullscreen nuke animation)

**Comment by Manly-Dwarf** [comment]
ok +1 I guess?

File: hippiestation/code/datums/mutations.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	owner.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/white(owner), SLOT_GLOVES) // this is purely for cosmetic purposes incase they aren't wearing anything in that slot
 	owner.equip_to_slot_or_del(new /obj/item/storage/backpack/clown(owner), SLOT_BACK) // ditto
 
-/datum/mutation/human/cluwne/on_life(mob/living/carbon/human/owner)
+/datum/mutation/human/cluwne/on_life()
```
Comment (yoyobatty) [inline hippiestation/code/datums/mutations.dm:32]: why did you take out this line?

File: hippiestation/code/datums/mutations.dm [hunk 2]
```diff
@@ -55,7 +55,7 @@
 					"<span class='danger'>Your brain feels like it's being torn apart, and after a short while, you notice that you've become a cluwne!</span>")
 	flash_act()
 
-/datum/mutation/human/cluwne/on_life(mob/living/carbon/human/owner)
+/datum/mutation/human/cluwne/on_life()
```
Comment (yoyobatty) [inline hippiestation/code/datums/mutations.dm:58]: and this one?

File: hippiestation/code/datums/mutations.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/datums/mutations.dm:32]: `mutations.dm`, line 100:
```/datum/mutation/human/proc/on_life()```

File: hippiestation/code/datums/mutations.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/datums/mutations.dm:58]: `mutations.dm`, line 100:
```/datum/mutation/human/proc/on_life()```

**Comment by karmaisblackandbluepilled** [comment]
+1

**Comment by notoriousBIGGIOS** [comment]
+1

**Comment by ghost** [comment]
image
image


ok then

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**